SPECS/cloud-hypervisor/cloud-hypervisor.signatures.json

```diff
@@ -1,7 +1,7 @@
 {
  "Signatures": {
-  "cloud-hypervisor-28.0-cargo.tar.gz": "caa7a53edb22326df4ab9594deaaa7a686324cf80f76bce65b0a86a4b524a15e",
-  "cloud-hypervisor-28.0.tar.gz": "c04bd4b511a142bec8fb97181b121d8d0623c16188867e860502e936cb751948",
-  "config.toml": "e823c53144d9c88262f033560e988e2431a2f0b4a815ff0be8740f0c0bb9312c"
+  "cloud-hypervisor-29.0-cargo.tar.gz": "15b013e359f8e578c67ff16840817882023f3547823bb797b67a9c03d8be1b1d",
+  "cloud-hypervisor-29.0.tar.gz": "80f235002d2fd61e5840772f4faf4877c87fff9c3f3522ce4a68317398a2ee3f",
```

Comment:
Looks like these two tar files aren't on the source tar mirror, please make sure they're uploaded.

Reply:
tar uploaded with name (cloud-hypervisor-29.0.tar.gz) matching what SPEC file expects https://dev.azure.com/mariner-org/mariner/_build/results?buildId=307099&view=logs&j=67da4c17-a18b-5c48-5cd0-dce235a264ea&t=aed30cd1-418f-54ed-5ec6-2a5143ce4f18&l=80

cargo tar uploaded: https://dev.azure.com/mariner-org/mariner/_build/results?buildId=307115&view=logs&j=67da4c17-a18b-5c48-5cd0-dce235a264ea&t=aed30cd1-418f-54ed-5ec6-2a5143ce4f18&l=78